### PR TITLE
:memo: add example reference. closes #82

### LINKED ISSUE #82: documentation request: consider adding to the #usage section an explicit example of integrating with Hapi server <eom>

### PR DESCRIPTION
**Pull request by cdaringe**
# problem statement

- usage xref can ease user search

# solution

- add xref

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Creates a new GoodConsole object with the following arguments:
 	- `utc` - boolean controlling Moment using [utc mode](http://momentjs.com/docs/#/parsing/utc/) or not. Defaults to `true`.
 	- `color` - a boolean specifying whether to output in color. Defaults to `true`.
 
+For an example of how to integrate good-console into a hapi project, please see [good example usage](https://github.com/hapijs/good#example-usage).
```
Comment (AdriVanHoudt) [inline README.md]: I think this would be better suited directly under the `Usage` title

Comment (arb) [inline README.md]: Do we really need this? This is just another link that can break when the good readme changes.

Comment (cdaringe) [inline README.md]: I think that depends on how much you want to help your users.  Given that READMEs are for helping users, I think it's fitting.  If the readme changes, no big deal, it takes you to the top of their README

Comment (AdriVanHoudt) [inline README.md]: You can also just point to the good docs instead of the specific example section

**Pull request closed without merging**